docs/scripts/plugin_documentation

```diff
@@ -3,7 +3,7 @@
 # The best default branch to use develop rather than master. As new repository do not have a branch called master any more and having main branch only.
 if [[ "${DOCBRANCH}" == "" ]]; then
     # NOTE: During release time we need to replace develop with actual released version
-	export DOCBRANCH=develop
+	export DOCBRANCH=1.9.1RC
```

Comment:
I assume just before the merge, after successful review of 1,9,1rc active readthedocs, we can make it main. and on main -> develop sync bring back to develop

Reply:
In the main branch, we are keeping DOCBRANCH as the latest RC version only, we will be changing DOCBRANCH value once it is in develop.